docsp-34265 - ef links (#137)
mongodb/docs-csharp: source/index.txt

driver versions in the :ref:`Upgrade Driver Versions <csharp-upgrade-driver>`
section.

Entity Framework Provider
-------------------------

> The MongoDB Entity Framework Provider is an object-relational mapper (ORM) that lets you
> use Microsoft's Entity Framework to work with MongoDB data in an object-oriented way.

Reviewer: suggestion:

```suggestion
The MongoDB Entity Framework Provider is an object-relational mapper (ORM) that lets you
use Microsoft's Entity Framework to work with MongoDB data. ORM provides an object-oriented interface for data management.
```

Author: good suggestion. i wasn't crazy about that sentence to begin with
